rpc/src/module/pool.rs

```diff
@@ -42,10 +57,23 @@ impl PoolRpcImpl {
 }
 
 impl PoolRpc for PoolRpcImpl {
-    fn send_transaction(&self, tx: Transaction) -> Result<H256> {
+    fn send_transaction(
+        &self,
+        tx: Transaction,
+        outputs_validator: Option<OutputsValidator>,
+    ) -> Result<H256> {
         let tx: packed::Transaction = tx.into();
         let tx: core::TransactionView = tx.into_view();
 
+        if let Err(e) = match outputs_validator {
+            Some(OutputsValidator::Default) | None => {
+                DefaultOutputsValidator::new(self.shared.consensus()).validate(&tx)
+            }
+            Some(OutputsValidator::Passthrough) => Ok(()),
+        } {
```

Comment:
IMO, returning the error reason would be better for users.

Reply:
Updated, please review again, thanks.